src/main/java/com/mindee/geometry/BoundingBoxUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.mindee.geometry;
+
+import java.util.DoubleSummaryStatistics;
+import java.util.List;
+
+public final class BoundingBoxUtils {
```

Comment:
This utils class should be moved to the utils package

Reply:
Will be not better to group class according to their domain instead of technicals terms ?
Is it not better to have little class instead of big one which have everything ?

Reply:
I would agree that it's best to keep them separate as the operations are likely very different.